lib/ui/color_constants.dart

```diff
@@ -1,7 +1,25 @@
 import 'dart:ui';
 
 class ColorConstants {
+  ColorConstants._();
+
   static const Color modesButtonUnpressedColor = Color(0xFFDFE0E4);
 
   static const Color modesButtonPressedColor = Color(0xFF5D657A);
+
+  static const Color pinCodeBottomSheetListTitleBg = Color(0xFFFFFFFF);
```

Comment:
Some point in the future I'd really like to see us moving towards using themes to manage this sort of stuff. Probably something for the backlog if it's not already there.